### PR TITLE
Notification for contest registration open to user and admin #7514

### PR DESCRIPTION
**Pull request by pabo99**
# Description

Added the notifications when user send request for registration for contest.

user sends request for registration
(contestant's view)

image
image

admin recieves request notification
(admin's view)

image

clicking on the notification takes them to the particular contest page for managing contestants
(admin's view)

image

if admin rejects the request contestent recive notification showing the request has been rejected.
(contestant's view)

image

if admin accepts the request contestent recive notification showing the request has been accepted.
(contestant's view)

image

clicking on the notifcaiton will take the contestant to the particular contest page
(contestant's view)

image

image

Fixes: #7514

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

File: frontend/server/src/Controllers/Contest.php [hunk 1]
```diff
@@ -4628,7 +4652,19 @@ public static function apiArbitrateRequest(\OmegaUp\Request $r): array {
...
         );
 
-        return ['status' => 'ok'];
+        $response = ['status' => 'ok'];
+
+        if (is_null($targetIdentity->user_id)) {
+            return $response;
+        }
+
+        \OmegaUp\Controllers\Notification::createForArbitrateRequest(
```
Comment (heduenas) [inline frontend/server/src/Controllers/Contest.php:4661]: Puedes renombrarlo a `createForCourseAccessRequest`?

File: frontend/templates/es.lang [hunk 2]
```diff
@@ -868,6 +868,9 @@ nominationsRangeHeader = "Reportes de problemas (del %(lowCount) al %(highCount)
 notProblemLevelTag = "La etiqueta enviada no es una etiqueta de nivel."
 notificationContestClarificationRequest = "Ha sido solicitada una nueva clarificación para el problema %(problemAlias) del concurso %(contestAlias), que administras."
 notificationContestClarificationResponse = "Su pedido de clarificación para el problema %(problemAlias) del concurso %(contestAlias), ha sido respondido."
+notificationContestRegisterRequest = "%(username) ha pedido registrarse para el concurso %(contestTitle) que tú manejas."
+notificationContestRegisterationAccepted = "Your registration request for contest %(contestTitle) has been **accepted**."
```
Comment (heduenas) [inline frontend/templates/es.lang:872]: Falta traducir este y el siguiente

File: frontend/templates/pt.lang [hunk 3]
```diff
@@ -868,6 +868,9 @@ nominationsRangeHeader = "Relatórios de problemas (de %(lowCount) a %(highCount
 notProblemLevelTag = "O tag enviado não é um tag de nível."
 notificationContestClarificationRequest = "Uma nova clarificação tem sido solicitada para o problema %(problemAlias) do concurso %(contestAlias), que você administra."
 notificationContestClarificationResponse = "Seu pedido de clarificação para o problema %(problemAlias) do concurso %(contestAlias), tem sido respondido."
+notificationContestRegisterRequest = "%(username) solicitou registro para o concurso %(contestTitle) que você gerencia."
+notificationContestRegisterationAccepted = "Your registration request for contest %(contestTitle) has been **accepted**."
```
Comment (heduenas) [inline frontend/templates/pt.lang:872]: Tambien en portugues falta traducir estos mensajes

File: frontend/server/src/Controllers/Contest.php (hunk 1 above)
Comment (pabo99) [reply frontend/server/src/Controllers/Contest.php]: Listo

File: frontend/templates/es.lang (hunk 2 above)
Comment (pabo99) [reply frontend/templates/es.lang]: Ups! Se me pasaron estas traducciones

File: frontend/templates/pt.lang (hunk 3 above)
Comment (pabo99) [reply frontend/templates/pt.lang]: Listo

**Review by heduenas — APPROVED** [review]

**Pull request merged**